src/TestProblem/Hydro/Plummer/FB_Plummer.cpp

```diff
@@ -87,7 +87,8 @@ extern void (*FB_End_User_Ptr)();
 // Return      :  Fluid, ParAtt
 //-------------------------------------------------------------------------------------------------------
 int FB_Plummer( const int lv, const double TimeNew, const double TimeOld, const double dt,
-                const int NPar, const int *ParSortID, real *ParAtt[PAR_NATT_TOTAL],
+                const int NPar, const long *ParSortID, real *ParAtt[PAR_NATT_TOTAL],
+//                const long NPar, const float *ParSortID, double *ParAtt[PAR_NATT_TOTAL],
```

Comment:
Remove it

Reply:
Oh, sorry, I forgot to delete the line for bug test :sweat: